Public/Src/Deployment/buildXL.dsc

```diff
@@ -62,19 +62,23 @@ namespace BuildXL {
                             }).exe
                         ]
                     },
-                    {
-                        subfolder: r`NinjaGraphBuilder`,
-                        contents: [
-                            importFrom("BuildXL.Tools").NinjaGraphBuilder.exe,
-                            importFrom("BuildXL.Tools.Ninjson").pkg.contents
-                        ]
-                    },
-                    {
-                        subfolder: r`CMakeRunner`,
-                        contents: [
-                            importFrom("BuildXL.Tools").CMakeRunner.exe,
-                        ]
-                    },
+                    ...(BuildXLSdk.Flags.deployExperimentalTools

```

Comment:
Nit: `...addIfLazy(BuildXLSdk.Flags.deployExperimentalTools, () => [ ...` to avoid evaluation completely.

Reply:
They should not be evaluated with if-then-else expression, right?

Reply:
AFAIK they are, thats why making it lazy saves some work!